go.mod

```diff
@@ -52,13 +52,13 @@ require (
 	gorm.io/driver/mysql v1.1.1
 	gorm.io/gorm v1.21.12
 	helm.sh/helm/v3 v3.7.1
-	k8s.io/api v0.22.2
+	k8s.io/api v0.23.5
```

Comment:
I'm certainly not opposed to bumping the k8s.io dependencies, but could you add your reasons why to the PR description/commit message? ("General upkeep" is a perfectly valid reason if that's the case, but just so we know what the implications are if we need to cherry-pick or otherwise move this commit around later.)

Reply:
The previous CI run was complaining about `go mod tidy` so I ran that which must have bumped this. I'm guessing it was required to reconcile the new kubectl package?

I'll add a comment